### ISSUE TITLE
Colon [:] button does not work in converter

### ISSUE THREAD

**Issue by trentj** [bestia-dev/new_date_time_units_and_formats#2]
At https://bestia.dev/new_date_time_units_and_formats/, if I choose either input format that contains a `:`, the button does nothing. I can't enter a time for conversion; instead I see the "unrecognized format" error.

I tried on an Android device and using Firefox on Linux with the same symptom.

**Comment by LucianoBestia** [comment]
Thank you trentj.
I corrected this now. The bug is corrected in v2021.205.1219
When I upload the new version, sometimes the PWA gets updated, but sometimes not. I don't know why yet. 
So it is important to know what version is actually on the local device. In the Hamburger menu "About" page I added the version of the PWA.
I also disabled the double-click to zoom. It was distracting for a "full screen" PWA. Also, fast click/touch on buttons are now always single-clicks and the typing experience is faster and better.
